src/perf/lib/quicmain.cpp

```diff
@@ -65,9 +60,9 @@ QuicMainStart(
 
     if (IsValue(TestName, "Throughput")) {
         if (ServerMode) {
-            TestToRun = new ThroughputServer{SelfSignedConfig};
+            TestToRun = new ThroughputServer(SelfSignedConfig);
         } else {
-            TestToRun = new ThroughputClient{};
```

Comment:
I would prefer to not get rid of these. The errors if something changes is a bit cleaner.

Reply:
Either way it's just calling the default constructor. I don't see how errors would be different. Personally, I prefer no parenthesis if there are no arguments.

Reply:
I guess. I'm just always a fan of being more explicit, But its not a big deal.